docs/specs/toc.yml

```diff
@@ -822,3 +822,39 @@ outputs:
   build.manifest:
   build.log: |
     ["error","yaml-duplicate-key","(Line: 3, Character: 3) - (Line: 3, Character: 12): Key 'topicHref' is already defined, remove the duplicate key.","docs/TOC.yml"]
+---
+# The included toc' errors/warning should be assigned to itself, not its parent
+inputs:
+  docfx.yml: |
+    content:
+      include:
+        - docs/TOC.md
+  docs/TOC.md: |
+    # [Reference TOC File](f1/TOC.yml)
+  docs/f1/TOC.yml: |
+    - name: Invalid Index Reference
+      href: index.md
+    - name: Invalid TocHref Reference
+      tocHref: index.md
+outputs:
+  docs/TOC.json: |
```

Comment:
nit: if the test is to verify logs, you can omit content of this file for clarity.